factory/processing/processComponentsHandler_test.go

```diff
@@ -211,11 +212,12 @@ func TestManagedProcessComponents_Create(t *testing.T) {
 		t.Parallel()
 
 		args := createMockProcessComponentsFactoryArgs()
+		args.GenesisMetaBlockChecker = processComp.NewSovereignGenesisMetaBlockChecker()
 		args.RunTypeComponents = componentsMock.GetSovereignRunTypeComponents()
-		processComponentsFactory, _ := processComp.NewProcessComponentsFactory(args)
-		managedProcessComponents, _ := processComp.NewManagedProcessComponents(processComponentsFactory)
-		_ = managedProcessComponents.Create()
-
+		processComponentsFactory, err := processComp.NewProcessComponentsFactory(args)
+		managedProcessComponents, err := processComp.NewManagedProcessComponents(processComponentsFactory)
```

Comment:
These lines will cause a linter `ineffectual assignment`.
If you added those errors, you should check they are nil